engine/consensus/dkg/messaging_engine.go

```diff
@@ -13,6 +15,13 @@ import (
 	"github.com/onflow/flow-go/network"
 )
 
+// RETRY_MAX is the maximum number of times the engine will attempt to broadcast
+// a message
+const RETRY_MAX = 5
+
+// RETRY_MILLISECONDS is the number of milliseconds to wait between retries
```

Comment:
Maybe this is more accurate since the wait between retries is not constant
```suggestion
// RETRY_MILLISECONDS is the number of milliseconds to wait between the two first tries
```